di-ipv-queue-stub/README.md

```diff
@@ -1,83 +1,34 @@
-### Enqueue messages
+# IPV Queue stub
+
 This will set up an API gateway in front of a lambda and push messages onto an SQS queue.
+Currently used for testing F2F asynchronous VC return.
+
+## Sending messages
+
 The format of the POST request to the API gateway should look like
-```
-{
-    "queueName": "stubQueue_AnyQueueName",
-    "queueEvent": <This can be  any shape - it will be sent to the queue as a string>,
-    "delaySeconds": Optional number of seconds to wait before porting  to the queue
-}
-```
-or
-```
-{
-    "queueName": "stubQueue_AnyQueueName",
-    "sub": <the sub field in the queue message body>
-    "state": <the state field in the queue message body>
-    "issuer": <name of the issuer you would like added to the queue>
-    "customClaims" : <the vc you want this to put onto the queue>
-    "privateSiginingKeyId" : <a key ID that the lamda can access, one is provided by the stack>,
-    "delaySeconds": Optional number of seconds to wait before porting  to the queue
-}
```

Comment:
The stub currently has a bunch of functionality for signing payloads sent to it and constructing VC JWTs.

As far as I am aware, the only client for the queue stub is the F2F stub, which constructs its own VC JWTs. If this is true, we can rip out all of this additional functionality and keep the queue stub simple.